viewer/packages/api/src/api-helpers/Image360ApiHelper.ts

```diff
@@ -70,9 +76,9 @@ export class Image360ApiHelper {
   );
 
   private readonly _activeCameraManager: ProxyCameraManager;
-  private readonly _image360Navigation: StationaryCameraManager;
+  private readonly _stationaryCameraManager: StationaryCameraManager | undefined = undefined;
```

Comment:
In TypeScript, we do not need to explicitly assign `undefined` value while creating the variable, it automatically assigns `undefined` value when created!

Reply:
Fixed